asoc_automation_iast/AsocUtils.py

```diff
@@ -179,8 +190,9 @@ def create_app(token, app_name, asset_group, host=ASOC_API, retries=0):
 # request URL : GET https://cloud.appscan.com/api/V4/Apps
 #     headers: "Authorization=Bearer <token>"
 #     params: "$filter: Id eq <appId>"
-def get_app_name_by_id(app_id, token, host=ASOC_API):
-    url = url_join(host, "/Apps")
+def get_app_name_by_id(app_id, token, host=None):
+    host = get_host(host)
+    url = url_join(host, ASOC_HOST, "/Apps")

```

Comment:
```suggestion
    url = url_join(get_host(host), ASOC_HOST, "/Apps")
```
could make it more compact

Reply:
done.